src/main/java/gift/controller/ProductController.java

```diff
@@ -0,0 +1,47 @@
+package gift.controller;
+
+import gift.model.Product;
+import gift.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+public class ProductController {
+
+    @Autowired
+    private ProductService productService;
+
+    @GetMapping("/")
+    public String viewHomePage(Model model) {
+        model.addAttribute("listProducts", productService.getAllProducts());
```

Comment:
```suggestion
        model.addAttribute("products", productService.getAllProducts());
```
복수형으로 쓰게 되면서 실제 Collection이라는걸 나타낼수 있을거 같아요~

그래서 이름을 이렇게 해도 괜찮을거 같습니다~